feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1002 @@
+	aggr.Type = oc.IETFInterfaces_InterfaceType_ieee8023adLag
+
+	for _, port := range portList {
+		i := d.GetOrCreateInterface(port.Name())
+		i.GetOrCreateEthernet().AggregateId = ygot.String(aggID)
+
+		i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+
+		if deviations.InterfaceEnabled(dut) {
+			i.Enabled = ygot.Bool(true)
+		}
+	}
+	p := gnmi.OC()
+	fptest.LogQuery(t, fmt.Sprintf("%s to Update()", dut), p.Config(), d)
+	gnmi.Update(t, dut, p.Config(), d)
+}
+
+// clearAggregate delete any previously existing members of aggregate.
+func clearAggregate(t *testing.T, dut *ondatra.DUTDevice, aggID string, agg *aggPortData, portList []*ondatra.Port) {
+	// Clear the aggregate minlink.
+	gnmi.Delete(t, dut, gnmi.OC().Interface(aggID).Aggregation().MinLinks().Config())
+	// Clear the members of the aggregate.
+	for _, port := range portList {
+		resetBatch := &gnmi.SetBatch{}
+		gnmi.BatchDelete(resetBatch, gnmi.OC().Interface(port.Name()).Ethernet().AggregateId().Config())
+		gnmi.BatchDelete(resetBatch, gnmi.OC().Interface(port.Name()).ForwardingViable().Config())
+		resetBatch.Set(t, dut)
+	}
+}
+
+// configureDUTBGP configure BGP on DUT
+func configureDUTBGP(t *testing.T, dut *ondatra.DUTDevice, aggIDs []string) {
+	t.Helper()
+
+	d := &oc.Root{}
+	ni := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
+	niProto := ni.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	bgp := niProto.GetOrCreateBgp()
+
+	global := bgp.GetOrCreateGlobal()
+	global.RouterId = ygot.String(dutLoopback.IPv4)
+	global.As = ygot.Uint32(asn)
+	global.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)
+	global.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).Enabled = ygot.Bool(true)
+	pgName := "BGP-PEER-GROUP1"
+	pg := bgp.GetOrCreatePeerGroup(pgName)
+	pg.PeerAs = ygot.Uint32(asn)
+	if deviations.RoutePolicyUnderAFIUnsupported(dut) {
+		rpl := pg.GetOrCreateApplyPolicy()
+		rpl.SetExportPolicy([]string{acceptRoutePolicy})
+		rpl.SetImportPolicy([]string{acceptRoutePolicy})
+	} else {
+		af4 := pg.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST)
+		af4.Enabled = ygot.Bool(true)
+		rpl := af4.GetOrCreateApplyPolicy()
+		rpl.SetExportPolicy([]string{acceptRoutePolicy})
+		rpl.SetImportPolicy([]string{acceptRoutePolicy})
+
+		af6 := pg.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST)
+		af6.Enabled = ygot.Bool(true)
+		rpl = af6.GetOrCreateApplyPolicy()
+		rpl.SetExportPolicy([]string{acceptRoutePolicy})
+		rpl.SetImportPolicy([]string{acceptRoutePolicy})
+	}
+
+	for _, a := range []*aggPortData{agg1, agg2, agg3} {
+		bgpNbrV4 := bgp.GetOrCreateNeighbor(a.ateLoopbackV4)
+		bgpNbrV4.PeerGroup = ygot.String(pgName)
+		bgpNbrV4.PeerAs = ygot.Uint32(asn)
+		bgpNbrV4.Enabled = ygot.Bool(true)
+		bgpNbrV4T := bgpNbrV4.GetOrCreateTransport()
+		bgpNbrV4T.LocalAddress = ygot.String(dutLoopback.IPv4)
```

Comment:
README says that interface addresses are used for IBGP and there's only one session (two if you want to do dual-stack)

https://github.com/openconfig/featureprofiles/blob/70b8fbdd36d45f666695495cf843571b25bc55ad/feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md?plain=1#L47

Reply:
Changed configuration for BGP from Loopback to LAG interface IP.